front/app/containers/ProjectsShowPage/timeline/Timeline.tsx

```diff
@@ -245,17 +235,37 @@ const Timeline = memo<Props>(({ projectId, className }) => {
     []
   );
 
-  if (
-    !isNilOrError(locale) &&
-    !isNilOrError(currentTenant) &&
-    !isNilOrError(phases) &&
-    phases.length > 0
-  ) {
+  const handleTabListOnKeyDown = (e: KeyboardEvent) => {
+    const arrowLeftPressed = e.key === 'ArrowLeft';
+    const arrowRightPressed = e.key === 'ArrowRight';
+
+    if ((arrowLeftPressed || arrowRightPressed) && !isNilOrError(phases)) {
+      const currentPhaseIndex = phases.indexOf(selectedPhase);
+
+      if (arrowRightPressed) {
+        // if we're at the end of the timeline, go to start (index 0),
+        // otherwise on to the right (index + 1)
+        const selectedPhaseIndex =
+          currentPhaseIndex === phases.length - 1 ? 0 : currentPhaseIndex + 1;
+        setSelectedPhase(phases[selectedPhaseIndex]);
+        tabsRef.current[selectedPhaseIndex].focus();
+
+        // Move left
+      } else if (arrowLeftPressed) {
+        // if we're at the beginning of the timeline, go to end (array length - 1),
+        // otherwise on to the left (index - 1)
+        const selectedPhaseIndex =
+          currentPhaseIndex === 0 ? phases.length - 1 : currentPhaseIndex - 1;
+        setSelectedPhase(phases[selectedPhaseIndex]);
+        tabsRef.current[selectedPhaseIndex].focus();
+      }
+    }
+  };
```

Comment:
Implemented similar to https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role.

The timeline, once focused, can be navigated using the left/right arrow. Then on tab, you go to the corresponding panel. On shift + tab, you go back to the timeline. MDN's implementation is much better than what we had. Navigating the timeline wasn't great for keyboard users.